packages/frontend/src/pages/SqlRunner.styles.ts

```diff
@@ -28,6 +28,7 @@ export const SqlCallout = styled(Callout)`
     margin-top: 20px;
 `;
 
+// @ts-ignore
 export const StyledTabs = styled(Tabs)`
```

Comment:
this one says that TableProps are not exposed or something right?

Reply:
yeah, blueprint types are not fully compatible with styled-components.